pkg/api/store/node/store.go

```diff
@@ -64,4 +68,5 @@ func (n nodeStore) Update(apiContext *types.APIContext, schema *types.Schema, da
 func format(data map[string]interface{}) {
 	data["desiredNodeLabels"] = data["labels"]
 	data["desiredNodeAnnotations"] = data["annotations"]
+	data["desiredNodeUnschedulable"] = map[string]bool{"value": convert.ToBool(data["unschedulable"])}
```

Comment:
desiredNodeUnschedulable should be set by cordon/uncordon action directly. `unschedulable` is k8s field, we should not manipulate it using our api